File: help/eng_gb/exercicios_da_ficha_9/Ficha09_005/Ficha09_005.js
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("Inferior limit : " );// print text
    inf = parseInt(prompt("Inferior limit : " ,"0") );
    
    document.write(inf + "\n");// print input 
    document.write("Upper limit : " );// print text
    sup = parseInt(prompt("Upper limit : " ,"0") );
    
    document.write(sup + "\n");// print input 
    while(inf <= sup ){
        if(inf % 2 != 0 ){
            document.write(inf + " " );
        }
        inf++;
    }
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Sat, 2022 Oct 01 00:11

File: help/eng_gb/exercicios_da_ficha_9/Ficha09_005/Ficha09_005.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha09_005 {

    //Main Function
    static void Main() {
        Console.Write("Inferior limit : " );
        int inf = int.Parse(Console.ReadLine());
        Console.Write("Upper limit : " );
        int sup = int.Parse(Console.ReadLine());
        while(inf <= sup ){
            if(inf % 2 != 0 ){
                Console.Write(inf + " " );
            }
            inf++;
        }
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Sat, 2022 Oct 01 00:11
